src/Form/AuthenticationForm.php

```diff
@@ -597,4 +500,155 @@ public function ajaxCallback(array $form, FormStateInterface $form_state): array
     return $form;
   }
 
+  /**
+   * Creates a new auth key stored in a file.
+   *
+   * @return \Drupal\key\KeyInterface
+   *   A new auth key.
+   *
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   * @throws \Drupal\key\Exception\KeyValueNotSetException
+   */
+  protected function generateNewAuthKey() {
+    // Create a new key name.
+    $new_key_id = 'apigee_edge_connection_default';
+    $file_path = "private://{$new_key_id}_apigee_edge.json";
```

Comment:
`apigee_edge_connection_default_apigee_edge.json` isn't it a weird name for this file? Is the apigee_edge suffix really necessary?

Reply:
Change to `private://{$new_key_id}.json`.